portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -14,19 +14,30 @@
 
 package com.google.sps.servlets;
 
+import java.util.ArrayList;
+import java.util.List;
 import java.io.IOException;
 import javax.servlet.annotation.WebServlet;
 import javax.servlet.http.HttpServlet;
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
 
 /** Servlet that returns some example content. TODO: modify this file to handle comments data */
-@WebServlet("/data")
-public class DataServlet extends HttpServlet {
+@WebServlet("/literature")
+public final class DataServlet extends HttpServlet {
 
+  private List<String> BookRecs;
+  @Override
+  public void init() {
+    BookRecs = new ArrayList<>();
+    BookRecs.add("The Thread by Victoria Hislop");
+    BookRecs.add("The Hunchback of Notre Dame by Victor Hugo");
+    BookRecs.add("The Brothers Karamazov by Fyodor Dostoevsky");
```

Comment:
just a btw, but another way of doing this is 
`new ArrayList<>(Arrays.asList(element 1, element 2,...))`